sdk/core/azure-core/src/main/java/com/azure/core/util/tracing/Tracer.java

```diff
@@ -170,20 +170,6 @@
      */
     Context setSpanName(String spanName, Context context);
 
```

Comment:
Since we've GA'd azure-core, wouldn't removing this be a breaking API change? @JonathanGiles

Reply:
Connie is right - this is breaking API.

Reply:
If this API is no longer required, and will never be required, we should consider deprecating it - but we can't just go straight to removing it.